lib/translator/translation.cc

```diff
@@ -0,0 +1,82 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "common.h"
+
+namespace translator {
+
+BeginResponse Translation::Begin(absl::Span<const uint8_t> scsi_cmd,
+                                 scsi_defs::LunAddress lun) {
+  BeginResponse response = {};
+  response.status = ApiStatus::kSuccess;
+  if (this->pipeline_status != StatusCode::kUninitialized) {
```

Comment:
```suggestion
  if (pipeline_status != StatusCode::kUninitialized) {
```
Member variables are in scope; they don't need to be qualified unless they're shadowed by something. Similar issues below.